core/src/main/java/org/elasticsearch/index/mapper/AllFieldMapper.java

```diff
@@ -136,6 +140,9 @@ public AllFieldMapper build(BuilderContext context) {
                 Object fieldNode = entry.getValue();
                 if (fieldName.equals("enabled")) {
                     boolean enabled = TypeParsers.nodeBooleanValue(name, "enabled", fieldNode);
+                    if (enabled) {
```

Comment:
I think that would mean a log message is always written no matter what, since the default (not specifying) is still enabled in 5.x?